setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import setup, find_packages
+
+PACKAGENAME = "testingframework"
+DESCRIPTION = "Module for testing various interatomic potentials"
+AUTHOR = "Libatoms"
+AUTHOR_EMAIL = ""
+
+version = {}
+with open("version.py") as fp:
+    exec(fp.read(), version)
+
+setup(
+    name=PACKAGENAME,
+    packages=find_packages(),
+    version=version["__version__"],
+    description=DESCRIPTION,
+    long_description=open("README.md").read(),
```

Comment:
shall we read this in above just like the other parts? just being a little petty here, but `with ... as` is nicer for this, even though we can be pretty sure that the file will be closed properly like this as well.